protocols/raft/src/main/java/io/atomix/protocols/raft/proxy/impl/RaftProxyInvoker.java

```diff
@@ -280,7 +278,10 @@ public void fail() {
      * @param t The exception with which to fail the attempt.
      */
     public void fail(Throwable t) {
```

Comment:
Failing to sequence a failed request can lead to responses being blocked waiting for an earlier response that will never come.

Reply:
What's sequence in this context?